### PR TITLE
Update Data Collection notebook

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Updated the data collection notebook by adding a find_dotenv and pipfile for better imports.
Also started to explore the sample data set.

### CODE REVIEW

**Review by Shreyanand — COMMENTED** [review]
/lgtm Just adding our meeting discussion points:

- #11 move this PR towards second issue in the KR
- Add 10k rows if available
- Add groupby, value_counts with suite_id, test_id
- Can we find how frequently a test runs (hourly, daily, weekly etc.)?
- Save a copy of the sample dataset somewhere. Do we have a bucket?

**Comment by aakankshaduggal** [comment]
> /lgtm Just adding our meeting discussion points:
> 
> * #11 move this PR towards second issue in the KR
> * Add 10k rows if available
> * Add groupby, value_counts with suite_id, test_id
> * Can we find how frequently a test runs (hourly, daily, weekly etc.)?
> * Save a copy of the sample dataset somewhere. Do we have a bucket?

Sounds good, I have added a bigger data chunk and some EDA for this sample dataset. Please let me know what you think @Shreyanand

File: notebooks/dataset_collection.ipynb [hunk 1]
```diff
@@ -5,47 +5,42 @@
    "id": "da691d86",
```
Comment (Shreyanand) [inline notebooks/dataset_collection.ipynb:5]: <p>100k seems like a good sample size!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/openshift-tests-optimization/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (Shreyanand) [inline notebooks/dataset_collection.ipynb:5]: <p>Can we get the date range for this data? Like first and last date.</p><p>Also, It seems like there are multiple rows for the same date so a value count for date could help as well. How many entries do we see on 2022-11-19 and so on...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/openshift-tests-optimization/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (Shreyanand) [inline notebooks/dataset_collection.ipynb:5]: <p><em>missing ~5.8% values</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/openshift-tests-optimization/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (Shreyanand) [inline notebooks/dataset_collection.ipynb:5]: <p>What's the reasoning behind summing these columns for a suite?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/openshift-tests-optimization/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (aakankshaduggal) [reply notebooks/dataset_collection.ipynb:5]: <p>Okay, sounds good.</p>

Comment (aakankshaduggal) [reply notebooks/dataset_collection.ipynb:5]: <p>yeah, we can fetch the date from the created_at value. </p>

Comment (aakankshaduggal) [reply notebooks/dataset_collection.ipynb:5]: <p>Yup, we can actually bring this up when we meet with the Openshift team.</p>

Comment (aakankshaduggal) [reply notebooks/dataset_collection.ipynb:5]: <p>No specific reason, just playing around and seeing how the grouped df looks.</p>

[the remark this answers is not in the document]
Comment (aakankshaduggal) [reply notebooks/dataset_collection.ipynb:5]: <p>Ah, yes. This is something we can explore in a detailed notebook I feel. In this notebook I think we just learn more about the data and how it looks and what it entails for a deeper analysis.</p>

[the remark this answers is not in the document]
Comment (aakankshaduggal) [reply notebooks/dataset_collection.ipynb:5]: <p>Yes, definitely.</p>

**Pull request merged**